backend/packages/Upgrade/.env.example

```diff
@@ -60,7 +60,7 @@ MONITOR_PASSWORD=1234
 # Google Client Id
 #
 GOOGLE_CLIENT_ID=google_project_id
-DOMAIN_NAME = playpowerlabs.com
+DOMAIN_NAME =
```

Comment:
You should give example in comment for different variations. Updating it actually also removes the variation when specific domain is required.

Reply:
Okay. I will do that.